opentreemap/importer/errors.py

```diff
@@ -23,51 +26,55 @@
 #
 # Errors that are attributed to rows
 #
-INVALID_GEOM = (10, 'Longitude must be between -180 and 180 and '
-                'latitude must be betwen -90 and 90', True)
+INVALID_GEOM = (10, trans('Longitude must be between -180 and 180 and '
+                          'latitude must be betwen -90 and 90'), True)
 
-GEOM_OUT_OF_BOUNDS = (11, 'Geometry must be inside the map bounds', True)
+GEOM_OUT_OF_BOUNDS = (
+    11, trans('Geometry must be inside the map bounds'), True)
 
-EXCL_ZONE = (12, 'Geometry may not be in an exclusion zone', True)
+EXCL_ZONE = (12, trans('Geometry may not be in an exclusion zone'), True)
 
-INVALID_SPECIES = (20, 'Could not find species with the given '
-                   'scientific name.', True)
-DUPLICATE_SPECIES = (21, 'More than one species matches the given '
-                     'scientific name.', True)
+INVALID_SPECIES = (20, trans('Could not find species with the given '
+                             'scientific name.'), True)
+DUPLICATE_SPECIES = (21, trans('More than one species matches the given '
+                               'scientific name.'), True)
 
-INVALID_OTM_ID = (30, 'The given Open Tree Map ID does not exist '
-                  'in the system. This ID is automatically generated '
-                  'by Open Tree Map and should only be used for '
-                  'updating existing records', True)
+INVALID_OTM_ID = (30, trans('The given Open Tree Map ID does not exist '
+                            'in the system. This ID is automatically '
+                            'generated by Open Tree Map and should only '
+                            'be used for updating existing records'), True)
 
-FLOAT_ERROR = (40, 'Not formatted as a number', True)
-POS_FLOAT_ERROR = (41, 'Not formatted as a positive number', True)
-INT_ERROR = (42, 'Not formatted as an integer', True)
-POS_INT_ERROR = (43, 'Not formatted as a positive integer', True)
-BOOL_ERROR = (44, 'Not formatted as a boolean', True)
-STRING_TOO_LONG = (45, 'Strings must be less than 255 characters', True)
-INVALID_DATE = (46, 'Invalid date (must by YYYY-MM-DD', True)
+FLOAT_ERROR = (40, trans('Not formatted as a number'), True)
+POS_FLOAT_ERROR = (41, trans('Not formatted as a positive number'), True)
+INT_ERROR = (42, trans('Not formatted as an integer'), True)
+POS_INT_ERROR = (43, trans('Not formatted as a positive integer'), True)
+BOOL_ERROR = (44, trans('Not formatted as a boolean'), True)
+STRING_TOO_LONG = (45, trans('Strings must be less than 255 characters'), True)
+INVALID_DATE = (46, trans('Invalid date (must by YYYY-MM-DD)'), True)
```

Comment:
@RickMohr @maurizi Don't want to sweep this under the rug - I noticed that this was missing a closing `)` and no other message is structured that way, so I rolled this fix in.